CriteoPublisherSdk/Tests/Utility/Categories/Criteo+Testing.m

```diff
@@ -41,6 +41,8 @@
 NSString *const PreprodInterstitialAdUnitId = @"test-PubSdk-Interstitial";
 NSString *const PreprodNativeAdUnitId = @"test-PubSdk-Native";
 
+NSString *const VideoInterstitialAdUnitId = @"test-video";
```

Comment:
The ad unit name is define in CDB preprod stub which share creative for both PubSDK and PubTag. I think you should keep the PubSdk mention there.